client/data/marketplace/use-marketplace-reviews.ts

```diff
@@ -290,12 +290,31 @@ export const useCreateMarketplaceReviewMutation = () => {
 	} );
 };
 
-export const useUpdateMarketplaceReviewMutation = () => {
+export const useUpdateMarketplaceReviewMutation = ( {
+	productType,
+	slug,
+	page,
+	perPage,
+	author,
+	author_exclude,
+}: MarketplaceReviewsQueryProps ) => {
 	const queryClient = useQueryClient();
+	const queryKey: QueryKey = [
+		queryKeyBase,
+		productType,
+		slug,
+		author,
+		author_exclude,
+		page,
+		perPage,
+	];
 	return useMutation( {
 		mutationFn: updateReview,
 		onSuccess: () => {
-			queryClient.invalidateQueries( { queryKey: queryKeyBase } );
+			queryClient.invalidateQueries( { queryKey } );
+		},
```

Comment:
We need to research a better way to invalidate other queries related to this product, as the average rating is not being updated after we edit a review.

Reply:
Added this here: https://github.com/Automattic/dotcom-forge/issues/4897